flocker/node/agents/blockdevice.py

```diff
@@ -1575,7 +1586,7 @@ def _discover_raw_state(self):
         compute_instance_id = api.compute_instance_id()
         volumes = api.list_volumes()
         system_mounts = {
-            mount.blockdevice: mount.mountpoint
+            mount.mountpoint: mount.blockdevice
```

Comment:
I had to flip this because on some systems (like the vagrant dev box) bind mount actually causes the blockdevice to be mounted in two directories:

```
$ cd /tmp
$ mkdir a b
$ mount --bind a b
$ mkdir a/c
$ mount -t tmpfs tmpfs a/c
$ mount
....
tmpfs on /tmp/a/c type tmpfs (rw,relatime)
tmpfs on /tmp/b/c type tmpfs (rw,relatime)
```